README.md

```diff
@@ -5,86 +5,115 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/canonical/microceph/microceph)](https://goreportcard.com/report/github.com/canonical/microceph/microceph)
 [![Documentation Status](https://readthedocs.com/projects/canonical-microceph/badge/?version=latest)](https://canonical-microceph.readthedocs-hosted.com/en/latest/?badge=latest)
 
-<p align="center">
-<a href="https://snapcraft.io/microceph">MicroCeph</a> is snap-deployed Ceph with built-in clustering.
-</p>
-
 [![Get it from the Snap Store][snap-button]][snap-microceph]
 
-## Table of contents
 
-* [💡 Philosophy](#-philosophy)
-* [🎯 Features](#-features)
-* [⚡️ Quickstart](#%EF%B8%8Fquickstart)
-* [📖 Documentation](#-documentation)
-* [💫 Project & community](#-project--community)
-* [📰 License](#-license)
+MicroCeph is an opinionated orchestration tool for Ceph clusters at all scales.
+It reduces the complexity of deploying and managing clusters by simplifying various operations like, service placement,
+disk administration and remote replication via RESTful APIs and CLI commands.
+
+Available as a snap, MicroCeph is the easiest tool for administrators, developers, and hobbyists to manage clusters.
+
+## Installation
+
+MicroCeph has first-class support as a snap. On snap-ready systems, you can install it on the command line with:
```

Comment:
"MicroCeph has first-class support as a snap" doesn't parse quite right for me -- first-class support to me reads like there are other options too -- but thats not quite right, as MC is only delivered as a snap, so there's no other support neither second nor third class :-) 

Not sure of alternatives, maybe "MicroCeph offers first-class support through snaps"?

Wrt to the second sentence, as a user the first question that would come to mind would be "is my system snap-ready?". Maybe we could make `snap-ready` a link that explains which systems are suitable?

Reply:
@sabaini: haha I hear it now (first-class support).
I have added t[his link](https://snapcraft.io/docs/installing-snapd) from the Snap documentation.

Reply:
> @sabaini: haha I hear it now (first-class support). 

:-) 
Suggestions for alternatives @skoech ?

> I have added [this link](https://snapcraft.io/docs/installing-snapd) from the Snap documentation.

Thanks, think that works fine!

Reply:
> Suggestions for alternatives @skoech ?

I've just gone with your words: "MicroCeph is delivered as a snap..."